dev-requirements.txt

```
@@ -1,12 +1,12 @@
 black==24.10.0; python_version > "3.8"
-black==24.8.0; python_version <= "3.8"
+black==24.10.0; python_version <= "3.8"
```

```suggestion
black==24.8.0; python_version <= "3.8"
```

Need to retain this for backwards compat reasons.